Shared/Models/ForKidsItem.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ForKids.swift
+//  Movapp (iOS)
+//
+//  Created by Daryna Polevyk on 11.04.2022.
+//
+
+import Foundation
+
+struct ForKidsItem: Identifiable, Decodable {
+    var id = UUID()
+    
+#if DEBUG
```

Comment:
dle @zoul toto nemůžeme používat kvůli release tak prosím odebrat :)

Reply:
Ok, necham to tak

Reply:
ten debug prosím odeber, nebude fungovat release :)